pytest_asyncio/plugin.py

```diff
@@ -484,7 +485,19 @@ def event_loop(request: "pytest.FixtureRequest") -> Iterator[asyncio.AbstractEve
     """Create an instance of the default event loop for each test case."""
     loop = asyncio.get_event_loop_policy().new_event_loop()
     yield loop
-    loop.close()
+    # Cleanup code copied from the implementation of asyncio.run()
+    try:
+        asyncio.runners._cancel_all_tasks(loop)
```

Comment:
Calling private functions here is a bit sketch, but it's what I chose to do in my project instead of copy/pasting the entire implementation. Might want to go with the copy/paste approach though for this library.

Reply:
Sorry for the late response… There are a couple of ideas flying around, all of which address this issue in some way. That makes it hard to review your PR, so I put it off for too long.

I agree that calling `_cancel_all_tasks` might not be a good idea. Although we test with different CPython versions, `_cancel_all_tasks` is an implementation detail that could change in a patch release. Since we only test different minor releases (i.e. 3.7, 3.8, …) a change of `_cancel_all_tasks` could slip through unnoticed.

It would be best to replicate the behaviour of `_cancel_all_tasks` in pytest-asyncio. We cant't do a 1:1 copy and paste, though, because the CPython code is protected by copyright and under a different license than pytest-asyncio.

Reply:
I still stand by my last comment on the use of `_cancel_all_tasks`.